### ISSUE TITLE
Funding for Merge meeting #6

### ISSUE THREAD

**Issue by Santhosh1692** [ethereum-cat-herders/funding#205]
PR :[Merge Meeting #6](https://github.com/ethereum/pm/pull/344)
Amount: 50 DAI
Wallet: 0xEcebC7c42E3D16f70964d7d350FdFaE51ca5b5f6

**Comment by Santhosh1692** [comment]
@poojaranjan Please confirm that If I have raised the correct grant for this task.

**Comment by poojaranjan** [comment]
It should be 150 DAI

**Comment by poojaranjan** [comment]
Initiated mutisig tx 108 for #205 & #213. Waiting on approval & execution.

**Comment by Santhosh1692** [comment]
@poojaranjan Do I need to request a new PR for the remaining 100 or can the existing PR be updated?

**Comment by poojaranjan** [comment]
I have included that along with another issue. 
The funds issued is 300 DAI.
The transaction is successful! Please close the issue if everything is settled.

**Comment by Santhosh1692** [comment]
I have received the 300DAI, closing this.
